### PR TITLE
Refine MI2 story for Laundry

### PR DESCRIPTION
**Pull request by jespera**
Bum!

### CODE REVIEW

File: webgasdospela/MI2part1.core [hunk 1]
```diff
@@ -70,7 +70,8 @@ start() >> {hatch := Closed;
 { player.loc = LaundryShip } ?
     { hatch.sticked != True, hatch = Open } ?
         walkto(path) >> {player.loc := WoodtickLeft; hatch := Closed} & { "You hear the hatch close as you go back to the several-ships place" }
-    walkto(path) >> {player.loc := WoodtickLeft} & { "You are among several ships" }
+    { hatch.sticked != True, hatch != Open } ?
+        walkto(path) >> {player.loc := WoodtickLeft} & { "You are among several ships" }
```
Comment (zoren) [inline webgasdospela/MI2part1.core:74]: with this change you wouldn't be able to ```walkto(path)``` if ``` hatch.sticked = True```

Comment (jespera) [reply webgasdospela/MI2part1.core:74]: Oh! You're right. I'll update shortly.

Comment (zoren) [reply webgasdospela/MI2part1.core:74]: I don't think there's anything to update. I fixed the issue that was here previously, By removing a rule.

Comment (jespera) [reply webgasdospela/MI2part1.core:74]: Superseeded by #2

**Pull request closed without merging**